guides/user/canary/judge/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: single
+title:  "The canary judge&mdash;how does it work?"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+> Note: Automated canary analysis in Spinnaker is designed to support pluggable
+> judges. This document describes how the default judge (NetflixACAJudge) works.
+
+To assess the quality of a canary deployment against a baseline, metrics from
+both deployments are compared in order to check for significant degradation.
+This is done in two phases:
+
+* Metric collection (retrieval)
```

Comment:
Would mention (maybe later, not so important for the release) that this part of the process is not a part of the pluggable "judge" - that is, the judge is not responsible for retrieving metrics.

Reply:
Yeah, maybe we can change the overall title from '...canary judge...' to 'canary analysis...'.